lib/web/apiserver.go

```diff
@@ -2069,7 +2071,7 @@ func (h *Handler) siteSessionGenerate(w http.ResponseWriter, r *http.Request, p
 }
 
 type siteSessionsGetResponse struct {
-	Sessions []session.Session `json:"sessions"`
+	Sessions []session.Session `json:"trackers"`
```

Comment:
The only thing that hits this is the proxy web API, right? So we don't need to be concerned about old clients looking for the `sessions` key?

Reply:
Actually this was a mistake on my part, I meant to change the JSON name of the new response type for the new route types.

Reply:
But yes, the only accessor is the web api served by the proxy.